frontend/src/App.tsx

```diff
@@ -1,36 +1,93 @@
-import React, {Component} from 'react';
+import React, { Component } from 'react';
+import styled from 'styled-components';
 import logo from './logo.svg';
-import './App.css';
- 
-class App extends Component {
- 
-    state:any = {};
- 
-    componentDidMount() {
-        setInterval(this.hello, 250);
+import LandingPage from './views/Landing';
+import fetchHello from './api/hello';
+
+const StyledApp = styled.div`
+  text-align: center;
+  
+  .App-logo {
+    height: 40vmin;
+    pointer-events: none;
+  }
+  
+  @media (prefers-reduced-motion: no-preference) {
+    .App-logo {
+      animation: App-logo-spin infinite 20s linear;
     }
- 
-    hello = () => {
-        fetch('/hello')
-            .then(response => response.text())
-            .then(message => {
-                this.setState({message: message});
-            });
-    };
- 
-    render() {
-        return (
-            <div className="App">
-                <header className="App-header">
-                    <img src={logo} className="App-logo" alt="logo"/>
-                    <h1 className="App-title">{this.state.message}</h1>
-                </header>
-                <p className="App-intro">
-                    To get started, edit <code>src/App.js</code> and save to reload.
-                </p>
-            </div>
-        );
+  }
+  
+  .App-header {
+    background-color: #282c34;
+    min-height: 100vh;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    font-size: calc(10px + 2vmin);
+    color: white;
+  }
+  
+  .App-link {
+    color: #61dafb;
+  }
+  
+  @keyframes App-logo-spin {
+    from {
+      transform: rotate(0deg);
     }
+    to {
+      transform: rotate(360deg);
+    }
+  }
+`;
+
+type MyProps = {};
+
+type MyState = {
+  message: String,
+};
+
+class App extends Component<MyProps, MyState> {
+
+  constructor(props: MyProps) {
+    super(props);
+
+    this.state = { message: 'Loading' };
+  }
+
+  componentDidMount() {
+    setInterval(this.hello, 250);
+  }
```

Comment:
nit: Set `this.state` with the 'Loading' message as a class field, and replace `this.state = { message: 'Loading' };` with `this.hello()`.

Reply:
I might want to push back on this change for the following two reasons:

1. ESLint recommends initializing state in the constructor instead of as class fields, and similarly, based on a lot of online examples, it seems like the convention is to initialize state in the constructor (although to be fair I think both are equivalent). 
2. In general, it's recommended that API calls are put in the `componentDidMount` function rather than the constructor ([see here](https://stackoverflow.com/questions/44811983/calling-action-from-constructor-vs-a-life-cycle-method/44812019)). If the concern was making sure 'Loading' doesn't stay shown for so long, I added an initial call to `this.hello` in `componentDidMount` to hopefully mitigate that.

Definitely open to discussion on this though if you feel strongly the other way.

Reply:
That makes perfect sense, thanks for the response -- sounds great by me.